payment_payzen/i18n/fr.po

```diff
@@ -211,6 +211,258 @@ msgstr ""
+msgid "First amount to pay"
+msgstr "Premier montant à payer"
+
+#. module: payment_payzen
+#. odoo-python
+#: code:addons/payment_payzen/models/sale_order.py:0
+#, python-format
+msgid "Invalid payment provider for recurring payment: %s"
+msgstr "Prestataire de paiement non valide pour le paiement récurrent : %s"
+
+#. module: payment_payzen
+#. odoo-python
+#: code:addons/payment_payzen/models/payment_transaction.py:0
+#, python-format
+msgid ""
+"Payment transaction (%s) with sale order with recurring payment provider set, but provider does not support recurring payments: %s"
+msgstr ""
+"Transaction de paiement (%s) avec une commande de vente dont le fournisseur de paiements récurrents "
+"est défini, mais le fournisseur ne prend pas en charge les paiements récurrents : %s"
+
+#. module: payment_payzen
+#: model:ir.model.fields,field_description:payment_payzen.field_sale_order__recurring_first_payment_amount
+msgid "Recurring First Payment Amount"
+msgstr "Montant du premier paiement récurrent"
+
+#. module: payment_payzen
+#: model:ir.model.fields,field_description:payment_payzen.field_sale_order__recurring_payment_monthly_amount
+msgid "Recurring Payment Monthly Amount"
+msgstr "Montant mensuel paiement récurrent"
+
+#. module: payment_payzen
+#: model:ir.model.fields,field_description:payment_payzen.field_sale_order__recurring_payment_months
+msgid "Recurring Payment Months"
+msgstr "Mois de paiement récurrent"
+
+#. module: payment_payzen
+#: model:ir.model.fields,field_description:payment_payzen.field_sale_order__recurring_payment_provider_id
+msgid "Recurring Payment Provider"
+msgstr "Fournisseur de paiements récurrents"
+
+#. module: payment_payzen
+#: model:ir.model.fields,field_description:payment_payzen.field_sale_order__recurring_second_payment_amount
+msgid "Recurring Second Payment Amount"
+msgstr "Montant du deuxième paiement récurrent"
+
+#. module: payment_payzen
+#. odoo-python
+#: code:addons/payment_payzen/models/payment_transaction.py:0
+#, python-format
+msgid ""
+"Recurring payment transaction (%s) provider doesn't match sale order "
+"provider: %s"
+msgstr ""
+"Le fournisseur de l'opération de paiement récurrent (%s) ne correspond pas au "
+"fournisseur de l'ordre de vente : %s"
+
+#. module: payment_payzen
+#: model:ir.model.fields,field_description:payment_payzen.field_sale_order__recurring_second_payment_date
+msgid "Second payment date"
+msgstr "Deuxième date de paiement récurrent"
+
+#. module: payment_payzen
+#: model:ir.model.fields,field_description:payment_payzen.field_payment_provider__support_recurring
+msgid "Supports Recurring Payments"
+msgstr "Prise en charge des paiements récurrents"
+
+#. module: payment_payzen
+#. odoo-python
+#: code:addons/payment_payzen/models/payment_transaction.py:0
+#, python-format
+msgid ""
+"Unsupported recurring payment transaction (%s) with multiple sale orders: %s"
+msgstr ""
+"Transaction de paiement récurrent non prise en charge (%s) avec plusieurs ordres de vente : %s"
```

Comment:
Same here (I don't know if all payments are done in the SO directly but seeing the translations, it seems to)